src/containers/ProfilePage/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import ProfileContainer from './ProfileContainer';
+import NoMatch from '../../utils/NoMatch';
+
+const ProfilePage = () => (
+  <Switch>
+    <Route exact path="/profile/edit" component={ProfileContainer} />
+    <Route component={NoMatch} />
```

Comment:
Do we have to load `NoMatch` here too? Just curious.

Reply:
@queq1890 Yea, of course. What's happen if you try to access `/profile/laksdjflasdjf`?
we put `NoMatch` component here to solve this case.

Reply:
OK so we need no match for each <Switch>. Thanks!